tap-agent/src/agent/sender_allocation.rs

```diff
@@ -417,6 +421,7 @@ impl SenderAllocationState {
                 .unwrap_or(BigDecimal::from(0))
                 .to_string()
                 .parse::<u128>()?,
+            counter: res.count.unwrap_or(0).to_string().parse::<usize>()?,
```

Comment:
why do you need to_string().parse::<usize>()?

Reply:
tried doing just res.count.unwrap_or(0 as usize) but it would still be u64, got any suggestion?